Update README.md (#330)
mattermost/mattermost-handbook: contributors/onboarding/staff-onboarding-guide/README.md

# Staff Onboarding

## Contents

* [WIP: Staff Onboarding Guide](https://handbook.mattermost.com/people/general-onboarding.html#wip-staff-onboarding-guide)
  * [Welcome to Mattermost](https://handbook.mattermost.com/people/general-onboarding.html#welcome-to-mattermost)
  * [How to use our Handbook](https://handbook.mattermost.com/)
  * [Our Mission](https://handbook.mattermost.com/company/about-mattermost#mission)
  * [Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles)
  * [Products](https://docs.mattermost.com/overview/product.html)
  * [Company](https://handbook.mattermost.com/company/about-mattermost/business-model)
  * [General Onboarding Checklist](https://handbook.mattermost.com/contributors/onboarding)

>   * [Departmental Onboarding Checklists](https://handbook.mattermost.com/contributors/onboarding)

Reviewer: Seems that this should link to "Departmental Onboarding Checklists and Information" section

>   * [Departmental Onboarding Checklists](https://handbook.mattermost.com/contributors/onboarding)

Author: ```suggestion
  * [Departmental Onboarding Checklists](https://handbook.mattermost.com/people/general-onboarding.html#contents)
```

---

> description: 1% complete.

Reviewer 2: We can remove description

\[ \] Learn how to format your messages using Markdown by reviewing [this guide](https://community.mattermost.com/help/messaging).

\[ \] Take Mattermost’s end user training [here](https://academy.mattermost.com/p/end-user-onboarding). Feel free to share any feedback with your manager.

\[ \] Register your laptop and any other Mattermost-issued equipment [here](https://forms.gle/yBkZo36hzzo8dsbKA).

\[ \] Set up your email signature. [Here’s how](https://docs.google.com/document/d/1KNfyWl40S6LcpZ4lk7ntiBeB0HBiQKAHuCAPFW1J0Zo/edit).

\[ \] Activate your Office365 account and download it to your computer. Instructions [here](https://support.office.com/en-us/article/download-and-install-or-reinstall-office-365-or-office-2019-on-a-pc-or-mac-4414eaaf-0478-48be-9c42-23adc4716658?ui=en-US&rs=en-US&ad=US\#InstallSteps=Install\_on\_a\_Mac).

##### Days 2-5

\[ \] Share your GitHub username with @lynn.conway. You can, and are encouraged, to use your personal GitHub account. You may also create a GitHub account with your @mattermost email instead.

> \[ \] Connect GitHub and Mattermost via the instructions here: [&lt;jump to convo&gt;\]\(/core/pl/i4k6eke5t38gxxwrjtpeegbhwr\).

Author: ```suggestion
\[ \] Connect GitHub and Mattermost via the instructions [here](https://community.mattermost.com/core/pl/i4k6eke5t38gxxwrjtpeegbhwr).
```

* \[ \] Meet with new staff to review required documentation \(e.g. [I-9 documents](https://www.uscis.gov/i-9)\).

_First Week Checklist_

#### Instructions

1. Please copy each item and add an - to the left of the bracket and an x inside of the bracket you complete the item \(e.g. - \[x\]\).
2. **Please reply to this message \(hover over any message to reveal the reply button\) with the items you’ve completed**.

##### Day 1

\[ \] Share your bio with your manager. This will be posted in the [Welcome channel](https://community.mattermost.com/private-core/channels/welcome) in the private Staff team by your manager with the hashtag **\#newcolleague**.

\[ \] Accept the invitation to your OneLogin account and switch your Mattermost account to use OneLogin from **Account Settings > Security > Sign-in Method > Switch to SAML**. Instructions are found [here](https://docs.google.com/presentation/d/1FsfSr6qgtjY4aCo_UoL7FSChwvX3iLXuCFKJYselxBo/edit#slide=id.p4). 
- **Note:** You’ll find other apps, like LastPass and Zoom, there too.

> \[ \] Download the Mattermost Desktop App [here](https://about.mattermost.com/downloads) and log in to [https://community.mattermost.com](https://community.mattermost.com).

Reviewer: ```suggestion
\[ \] Download the Mattermost Desktop App [here](https://mattermost.com/download/#mattermostApps) and log in to [https://community.mattermost.com](https://community.mattermost.com).
```

\[ \] Download the [Mattermost Mobile App](https://mattermost.com/download/#mattermostApps) on your smartphone and log in to [https://community.mattermost.com](https://community.mattermost.com) using your OneLogin account.

\[ \] Join the channels listed [here](https://docs.mattermost.com/process/training.html\#channels) to get a feel for the company and to see how we communicate.

\[ \] Explore your Mattermost account settings, making note of your notification settings.
- **Tip:** While you’re learning to use Mattermost, set your messages to send on CTRL+ENTER, so that using ENTER will insert a new line and not send the message immediately.

> \[ \] Learn how to format your messages using Markdown by reviewing [this guide](https://community-daily.mattermost.com/help/messaging).

Reviewer: ```suggestion
\[ \] Learn how to format your messages using Markdown by reviewing [this guide](https://community.mattermost.com/help/messaging).
```

\[ \] Take Mattermost’s end user training [here](https://academy.mattermost.com/p/end-user-onboarding). Feel free to share any feedback with your manager.

\[ \] Register your laptop and any other Mattermost-issued equipment [here](https://forms.gle/yBkZo36hzzo8dsbKA).

\[ \] Set up your email signature. [Here’s how](https://docs.google.com/document/d/1KNfyWl40S6LcpZ4lk7ntiBeB0HBiQKAHuCAPFW1J0Zo/edit).

\[ \] Activate your Office365 account and download it to your computer. Instructions [here](https://support.office.com/en-us/article/download-and-install-or-reinstall-office-365-or-office-2019-on-a-pc-or-mac-4414eaaf-0478-48be-9c42-23adc4716658?ui=en-US&rs=en-US&ad=US\#InstallSteps=Install\_on\_a\_Mac).

##### Days 2-5

\[ \] Share your GitHub username with @lynn.conway. You can, and are encouraged, to use your personal GitHub account. You may also create a GitHub account with your @mattermost email instead.

> \[ \] Connect GitHub and Mattermost via the instructions here: [&lt;jump to convo&gt;\]\(/core/pl/i4k6eke5t38gxxwrjtpeegbhwr\).

Reviewer: The link here doesn't look correct

##### Week 3

* \[ \] Send new staff \(and manager\) a [group message](https://docs.google.com/document/d/1TX2pnJebl7Mi2-R5u3R6PsjX8YOMS54xcI0KJhh9_xI/edit#bookmark=kix.toi80hx08jzs) sharing the [org chart](https://mattermost.bamboohr.com/employees/orgchart.php) and [staff email list](https://docs.google.com/spreadsheets/d/1NQE0fkZgavMTrSSB1aPWg5hGRL182S6AGsa4ts4pWZ4/edit#gid=649832066) and describing how to view other staff members’ calendars to book meetings.

##### Week 4

* \[ \] Send new staff \(and manager\) an [email](https://docs.google.com/document/d/1TX2pnJebl7Mi2-R5u3R6PsjX8YOMS54xcI0KJhh9_xI/edit#bookmark=id.reex8djwhwfa) inviting new staff to create their [Mattermost avatar](https://docs.mattermost.com/process/training.html#mattermost-avatar).

## Manager Responsibilities

### First Day

* \[ \] Post new staff member’s bio to the [Welcome channel](https://community.mattermost.com/private-core/channels/welcome) using the hashtag **\#newcolleague**.

> * \[ \] Add new staff to relevant aliases \(e.g. [sales@mattermost.com](mailto:sales@mattermost.com). 

Reviewer: ```suggestion
* \[ \] Add new staff to relevant aliases \(e.g. [sales@mattermost.com](mailto:sales@mattermost.com)).
```

- **Note:** If you do not have manager access to the alias\(es\), please contact the [People team](mailto:people@mattermost.com).

## [Departmental Onboarding Checklists and Information](https://handbook.mattermost.com/people/general-onboarding.html#contents)

\[1% draft. Placeholder for links to Departmental checklists\]

* Exec onboarding
* R&D onboarding

> > * [Product team onboarding](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#onboarding)

Reviewer: Some of the bullet points here are not aligned
